src/github/credentials.ts

```diff
@@ -20,13 +20,15 @@ const AUTH_INPUT_TOKEN_CMD = 'auth.inputTokenCallback';
 
 export class CredentialStore {
 	private _octokits: Map<string, Octokit>;
+	private _logins: Map<string, string>;
 	private _configuration: VSCodeConfiguration;
 	private _authenticationStatusBarItems: Map<string, vscode.StatusBarItem>;
 
 	constructor(configuration: any,
 		private readonly _telemetry: ITelemetry) {
 		this._configuration = configuration;
 		this._octokits = new Map<string, Octokit>();
+		this._logins = new Map<string, string>();
```

Comment:
Octokit actually exposes `currentUser`, it's just not shown in our typings.

Reply:
Oh, lol. My bad, *I* put it on there in #554 😅

I guess I'd still suggest doing that, mostly bc it really seems like it should be there and we can probably pull it upstream.

Reply:
Thanks!